scripts/check_frontend_coverage_test.py

```diff
@@ -224,21 +218,26 @@ def test_check_coverage_changes_decrease(self):
             with fully_covered_tests_swap:
                 with self.assertRaisesRegexp(
                     SystemExit,
-                    r'\033\[1mfile.ts\033\[0m file is in the whitelist but its'
-                    ' coverage decreased. Make sure it is fully covered'
-                    ' by Karma unit tests.\n'):
+                    r'\033\[1mfile.ts\033\[0m seems to be fully covered!'
+                    ' Before removing it manually from the blacklist'
+                    ' in the file'
+                    ' scripts/check_frontend_test_coverage.py, please'
+                    ' make sure you\'ve followed the unit tests rules'
+                    ' correctly on:'
+                    ' https://github.com/oppia/oppia/wiki/Frontend-unit-tests-guide#rules\n'): # pylint: disable=line-too-long
                     check_frontend_coverage.check_coverage_changes()
 
+
```

Comment:
is this an expected change?

Reply:
Done.

Reply:
is it resolved?

Reply:
Sorry! Actually we should have two blank lines between functions, right?

Reply:
I think we use one newline between methods in a class and two newlines between classes. (Can you please check other files.)

@Hudda, I think we have a linter test for this, can you please check? if not, then I think we should have that test.

Reply:
+1 with @DubeySandeep, we should have linter test for this.

Reply:
@DubeySandeep @bansalnitish Added in #8423

Reply:
> I think we use one newline between methods in a class and two newlines between classes. (Can you please check other files.)
> 
> @Hudda, I think we have a linter test for this, can you please check? if not, then I think we should have that test.

Yeah, you're right, just fix it in all file